src/utils/rtl.js

```diff
@@ -9,6 +9,10 @@ const reSkew = /((^|\s)skew(x|y)?\()\s*(\-?[\d]+)(deg|rad|grad)(,\s*(\-?[\d]+)(d
 export default function rtl(muiTheme) {
   if (muiTheme.isRtl) {
     return (style) => {
```

Comment:
What if exclude `directionInvariant` from result style object. For example
```javascrip
return ({directionInvarian, ...style}) => {
   if (style.directionInvariant === true) {
      return style;
   }
...
```

It's not strict and both variants are ok